pages/deployment/[uuid].tsx

```diff
@@ -53,6 +51,9 @@ function isTabOption(value: string): value is TabOptions {
   return ['overview', 'compliance', 'build', 'exports', 'settings'].includes(value)
 }
 
+const isVersionDoc = (value: unknown): value is VersionDoc =>
```

Comment:
This will be used in a bunch of places, so should be extracted to a utility file.

Reply:
However, in all the cases I can think of, `value` isn't `unknown`, it's either `ObjectId | VersionDoc` and we want to distinguish between the two.

This is because each of our `interface Model{}` stuff either populate the version field (so they're the full version doc) or just leave it as the identifier.

So I think this function should be changed to something like.

```
const isVersionPopulated = (value: Types.ObjectId | VersionDoc): value is VersionDoc => {
  return value.createdAt !== undefined
}
```